bower.json

```diff
@@ -14,22 +14,52 @@
   "homepage": "https://github.com/PolymerElements/app-route",
   "private": true,
   "ignore": [],
+  "dependencies": {
+    "iron-location": "PolymerElements/iron-location#2.0-preview",
+    "polymer": "Polymer/polymer#^2.0.0-rc.1"
+  },
   "devDependencies": {
-    "paper-styles": "polymerelements/paper-styles#^1.0.13",
-    "paper-input": "polymerelements/paper-input#^1.1.2",
-    "web-component-tester": "^4.0.0",
-    "iron-component-page": "polymerelements/iron-component-page#^1.0.0",
-    "iron-test-helpers": "polymerelements/iron-test-helpers#^1.0.0",
-    "iron-pages": "PolymerElements/iron-pages#^1.0.7",
-    "paper-card": "PolymerElements/paper-card#^1.1.1",
-    "paper-icon-button": "polymerelements/paper-icon-button#^v1.0.0",
-    "paper-toggle-button": "polymerelements/paper-toggle-button#^v1.0.0",
-    "google-youtube": "GoogleWebComponents/google-youtube#^1.2.1",
-    "iron-demo-helpers": "PolymerElements/iron-demo-helpers#^1.2.0",
-    "paper-spinner": "PolymerElements/paper-spinner#^1.1.1"
+    "google-youtube": "e111077/google-youtube#2.0-preview",
```

Comment:
I've tried to merge google-apis (a dep of google-youtube) but got no response. See: https://github.com/GoogleWebComponents/google-apis/pull/79
After that, I can merge google-youtube